ui/src/ImageUpload.tsx

```diff
@@ -52,11 +55,20 @@ function ImageUpload(props: ImageProps) {
                     bottomText={props.bottomText}
                     topText={props.topText}
             />
-            <form onSubmit={handleSubmit}>
-                <label className="button" htmlFor="browsePhoto">Browse...</label>
-                <input type="file" name="image" className="button" id="browsePhoto" onChange={onChangeHandler}/>
-                <button className="button" type="submit">Upload Image</button>
-            </form>
+            <Form onSubmit={handleSubmit}>
+                <InputGroup>
+                    <Form.File 
+                        id="custom-file"
+                        label="Choose file"
```

Comment:
The Bootstrap file browser does not update "Choose file" with the filename once a file has been selected. React recommends the `bs-custom-file-input` package to do that, but I was not able to figure out how to make it work. Is it worth putting more time into figuring that out?

Reply:
For now, no.  We can circle back to it later, if there's time.